### DISCUSSION TITLE
Question about currency calculator2

### DISCUSSION THREAD

**Discussion by starguo1998** [D3nan/Tafe24S1T102_Ext#4] [Q&A] [answered]
What event handling mechanism should we use to handle button clicks for navigating to different calculator functionalities or exiting the application?

**Comment by SamuelTAFE** [comment, marked as answer]
Hi Xiaorui,
I just used the inbuilt click events in Visual Studio (e.g. double click button in form to generate event), as that's all I've learned how to do for XAML/C# so far. Hopefully this would be suitable for page navigation and/or exiting? 
If we're having issues with re-using code over between different click events, factoring the repeated code out into a separate function (to be called in each click event) could also help.

**Reply by starguo1998** [reply]
Hi Samuel,

Thank you for sharing your approach! Using the built-in click events in Visual Studio is indeed a common and effective way to handle button clicks in XAML/C#. It's a straightforward method that allows developers to quickly generate event handlers for buttons.
